### DISCUSSION TITLE
How can I make the script running periodically?

### DISCUSSION THREAD

**Discussion by aahnik** [aahnik/tgcf#4] [Q&A] [answered]
I want to forward new messages from a telegram channel to another telegram channel constantly.

Question by a Quora user.

**Comment by aahnik** [comment, marked as answer]
For a continuos running script, [contact me in Telegram](https://telegram.me/@aahnikdaw).

**Reply by Chaitanyatapse** [reply]
Hii, I want the script that forward messages in real-time. I want to contact you about this. But It says telegram account doesn't exits.

**Reply by aahnik** [reply]
my telegram username is "aahnikdaw" (https://telegram.me/aahnikdaw)

**Comment by mamin123** [comment]
Oh..this script will not forward message continuously ? I thought that is what it suppose to do ?

**Reply by aahnik** [reply]
this script forwards all the past messages of a channel or group. When you run it, it does the job and then quits.

And I have a seperate script that keeps running indefinately and forwards whenever there is a new message.

**Reply by abuslaev** [reply]
@aahnik Please tell me how to do seperate script like you?

**Reply by aahnik** [reply]
that was a private script, but that feature will be added here soon. Please read #43 (the live sync) 

Live sync means - whenever a new message arrives in a source chat, it will be forwarded to destination chat.

Past means forwarding all the existing messages.

new version will be released on August 01, with lots of features. 

please star the repo, and click on watch to get notified.
